nexus/db-model/src/multicast_group.rs

```diff
@@ -223,6 +225,18 @@ pub struct ExternalMulticastGroup {
     pub version_removed: Option<Generation>,
 }
 
+impl ExternalMulticastGroup {
+    /// DPD tag for switch configuration.
+    ///
+    /// Uses the group's UUID to ensure uniqueness across the group's lifecycle.
+    /// This prevents tag collision when a group name is reused after deletion
+    /// (important given implicit create/delete semantics). Both external and
+    /// underlay groups use the same tag for pairing.
+    pub fn dpd_tag(&self) -> String {
+        self.id().to_string()
```

Comment:
Having the multicast group address in the tag would probably be nice for someone looking at dpd state and trying to figure out where things are coming from and/or what this tag actually means.